components/overflow-group/overflow-group-mixin.js

```diff
@@ -28,8 +29,7 @@ export const OverflowGroupMixin = superclass => class extends LocalizeCoreElemen
 	static get properties() {
 		return {
 			/**
-			 * Use predefined classes on slot elements to set min and max slotted items to show
-			 * @type {boolean}
+			 * @ignore
```

Comment:
Not (at least initially) supporting autoShow in `filter-overflow-group`, so moving documentation to just be in `overflow-group`

Reply:
If they _do_ set `auto-show` on the `d2l-filter-overflow-group` what happens?  Chaos?

Reply:
haha no chaos, it does actually work as is. This made me realize I need to make a tweak to max height though since wrapping could technically happen with `auto-show` and a `d2l-button-group-show` on an end element.